tests/codegen/trailing_zeros.rs

```diff
@@ -0,0 +1,21 @@
+// compile-flags: -O
+
+#![crate_type = "lib"]
+
+// CHECK-LABEL: @trailing_zeros_ge
+#[no_mangle]
+pub fn trailing_zeros_ge(val: usize) -> bool {
+    // CHECK: %[[AND:.*]] = and i64 %val, 7
```

Comment:
This test is going to fail on 32-bit systems due to different usize. You could either use more wildcards in the checks, or use a type like u64 or u32 to avoid the issue.